eng/CredScanSuppression.json

```diff
@@ -38,6 +38,8 @@
         "pwd1",
         "azure_client_secret&scope=https%3A%2F%2Fcognitiveservices.azure.com%2F.default",
         "azure_client_secret&scope=https%3A%2F%2Fdigitaltwins.azure.net%2F.default",
+        "azure_client_secret",
```

Comment:
We should probably also collapse `__CLIENT_SECRET__` and `AZURE_CLIENT_SECRET` and use one or the other in our repo.

Reply:
Done